tests/dvslib/dvs_vlan.py

```diff
@@ -0,0 +1,94 @@
+from dvs_database import DVSDatabase
+
+class DVSVlan(object):
+    def __init__(self, adb, cdb, sdb, cntrdb, appdb):
+        self.asic_db = adb
+        self.config_db = cdb
+        self.state_db = sdb
+        self.counters_db = cntrdb
+        self.app_db = appdb
```

Comment:
Remove self.state_db,self.counters_db, self.app_db variable if you are not using those variables in the code.

Reply:
Nice observation. Note that this a library class. Here we just have an instance where test_port_dpb_system.py is using it. But if you look at my upstream PR(https://github.com/Azure/sonic-swss/pull/1242), where test_vlan.py is also using this class, it will need other DBs.

Reply:
Awesome. got it.